target_postgres/connector.py

```diff
@@ -92,80 +92,34 @@ def interpret_content_encoding(self) -> bool:
         """
         return self.config.get("interpret_content_encoding", False)
 
-    def prepare_table(  # type: ignore[override]
```

Comment:
Git got really confused al diffing here. I deleted completely the method `prepare_table()` and created a new one `get_table_from_metadata()`